proto/src/determined/api/v1/experiment.proto

```diff
@@ -815,6 +815,8 @@ message SearchExperimentsRequest {
   int32 offset = 2;
   // How many results to show
   int32 limit = 3;
+  // Filter expression
```

Comment:
i think we should add more info for this param such as available operator and examples because user wouldnt know the syntax for filter.